src/app/components/email-validation/index.html

```diff
@@ -1,27 +1,115 @@
-    </sky-demo-page-property>
-  </sky-demo-page-properties>
+<div class="sky-margin-stacked-content-section">
+    <sky-demo-page-example>
+        <sky-email-validation-demo></sky-email-validation-demo>
+    </sky-demo-page-example>
+</div>
+    
+<sky-tabset id="content-tabs">
+	
+	<sky-tab tabHeading="Design">
+        
+        <sky-demo-page-content sectionHeading="Usage">
+            <h3 class="sky-text-success">Use when</h3>
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                    <p>Use email validation when users must enter valid email addresses.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/email-validation/email-validation-usage-1.png" />
+                    <div class="caption caption-success sky-text-success">Do validate email addresses that users will store as data.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+                
+            </sky-docs-visual-example>
+        
+            <h3 class="sky-text-danger">Don't use when</h3>
+            <sky-docs-visual-example>
+                <sky-docs-visual-example-text>
+                 <p>Don’t use email validation when users can enter multiple types of data in an input field.</p>
+                </sky-docs-visual-example-text>
+                
+                <sky-docs-visual-example-image>
+                    <div>
+                    <img src="~/assets/img/guidelines/email-validation/email-validation-usage-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't validate email addresses in search fields or filters.</div>
+                    </div>
+                </sky-docs-visual-example-image>
+            </sky-docs-visual-example>
+        
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Anatomy">
+            <sky-docs-anatomy imagePath="~/assets/img/guidelines/email-validation/email-validation-anatomy.png">
+                <sky-docs-anatomy-item anatomyNumber="1">Input field</sky-docs-anatomy-item>
+                <sky-docs-anatomy-item anatomyNumber="2">Status indicator (danger)</sky-docs-anatomy-item>
+            </sky-docs-anatomy>
+        </sky-demo-page-content>
+        
+        <sky-demo-page-content sectionHeading="Behavior & states">
+            <div class="sky-text-block">
+                <p>The email validation directive applies its logic when the input field loses focus and does not trigger error states while users type. After the field enters an error state, it remains in that state even after regaining focus and does not change until the content matches a valid email format.</p>
```

Comment:
In the second sentence, I'm inclined to switch it to, "After the field enters an error state, it remains in that state until the content matches a valid email format."

I'm not sure how important the bit about regaining focus is, but I think it makes it a little clearer when it's shortened.